core/whistle-1.0.0/src/wh_call_response.erl

```diff
@@ -38,6 +38,12 @@
 send(CallId, CtrlQ, Code) ->
     send(CallId, CtrlQ, Code, 'undefined').
 
+send(Call, Code, Cause, Media)
+  when element(1, Call) =:= 'whapps_call' ->
```

Comment:
I'd prefer to test CallId for is_binary/1 and have the second clause be `send(Call,...`